### PR TITLE
Update model in Object Detection sample from YoloV3 to YoloV4

### PR DESCRIPTION
**Pull request by sspintel**

### CODE REVIEW

File: examples/TF_1_x/object_detection_sample.py [hunk 1]
```diff
@@ -270,10 +168,15 @@ def load_labels(label_file):
...
+    anchors = get_anchors(anchor_file)
     input_name = "import/" + input_layer
-    output_name = "import/" + output_layer
     input_operation = graph.get_operation_by_name(input_name)
-    output_operation = graph.get_operation_by_name(output_name)
+    output_operation = [
+        graph.get_operation_by_name("import/" + output_name)
+        for output_name in output_layer
+    ]
+    print(input_operation, output_operation)
```
Comment (lganesan-intel) [inline examples/TF_1_x/object_detection_sample.py:179]: @sspintel This print statement can be removed

**Review by lganesan-intel — COMMENTED** [review]
Changes look good. 
Readme and [colab sample for object detection](https://github.com/openvinotoolkit/openvino_tensorflow/blob/releases/1.1.0/docs/cloud_instructions/Colab_OpenVINO_TensorFlow_object_detection_example.ipynb) needs to be updated to use YoloV4 instead of Yolov3.

**Review by ck-intel — APPROVED** [review]
changes looks fine.

**Review by ck-intel — APPROVED** [review]
Changes looks fine.

**Pull request merged**